### ISSUE TITLE
Fail to open Resident Evil 2 via Lutris

### ISSUE THREAD

**Issue by laichiaheng** [lutris/lutris#2240]
**Describe the bug**

I can't open the game since I switched to Ryzen7 3700X.

**Expected behavior**
Open the game like before

**Current behavior**

The game won't open

**Steps to reproduce**

open the game

**Lutris debugging output**

The terminal output of `lutris -d`.

```
INFO     2019-07-20 21:39:09,424 [application.do_command_line:225]:Running Lutris 0.5.2.2
INFO     2019-07-20 21:39:09,424 [startup.check_driver:60]:Using X.Org
INFO     2019-07-20 21:39:09,424 [startup.check_driver:65]:Running Mesa driver 19.1.2 on Radeon RX 580 Series (POLARIS10, DRM 3.32.0, 5.2.1-1-MANJARO, LLVM 8.0.0) (0x67df)
INFO     2019-07-20 21:39:09,424 [startup.check_driver:75]:GPU: 1002:67DF 1043:0521 using amdgpu drivers
Xlib:  extension "NV-GLX" missing on display ":1".
INFO     2019-07-20 21:39:09,468 [startup.check_vulkan:129]:Vulkan is supported
INFO     2019-07-20 21:39:09,469 [dxvk.get_dxvk_versions:24]:Updating DXVK versions
DEBUG    2019-07-20 21:39:09,587 [store.get_missing_media:182]:Requesting missing icons from API for melissa-k-and-the-heart-of-gold-collectors-edition, particula, doki-doki-literature-club, anoxemia, murderous-pursuits
DEBUG    2019-07-20 21:39:09,588 [http.get:68]:GET https://lutris.net/api/games
INFO     2019-07-20 21:39:09,592 [lutriswindow.toggle_connection:562]:Connected to lutris.net as s9209122222
DEBUG    2019-07-20 21:39:09,593 [http.get:68]:GET https://lutris.net/api/runtime
DEBUG    2019-07-20 21:39:09,597 [http.get:68]:GET https://lutris.net/api/games/library/s9209122222
DEBUG    2019-07-20 21:39:10,895 [api.get_game_api_page:132]:Loaded 5 games from page 1

(lutris:7414): Gtk-CRITICAL **: 21:39:11.680: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.680: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.680: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.680: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.681: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.681: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.681: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.681: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.681: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.681: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.682: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.682: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.683: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.683: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.683: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.683: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.683: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.683: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.684: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.684: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.685: gtk_tree_model_filter_real_unref_node: assertion 'elt->ref_count > 0' failed

(lutris:7414): Gtk-CRITICAL **: 21:39:11.685: gtk_tree_model_sort_real_unref_node: assertion 'elt->ref_count > 0' failed
DEBUG    2019-07-20 21:39:13,911 [application.launch:339]:Launching RESIDENT EVIL 2 / BIOHAZARD RE:2 (winesteam) (139620673181016)
DEBUG    2019-07-20 21:39:13,917 [game.play:269]:Prelaunching Winesteam runner (LutrisConfig(level=game, game_config_id=resident-evil-2-biohazard-re2-steam-1563550967, runner=winesteam))
INFO     2019-07-20 21:39:14,001 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/system32/dxgi with DXVK version
INFO     2019-07-20 21:39:14,001 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/system32/d3d11 with DXVK version
INFO     2019-07-20 21:39:14,001 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/system32/d3d10core with DXVK version
INFO     2019-07-20 21:39:14,001 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/system32/d3d10_1 with DXVK version
INFO     2019-07-20 21:39:14,001 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/system32/d3d10 with DXVK version
INFO     2019-07-20 21:39:14,002 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/syswow64/dxgi with DXVK version
INFO     2019-07-20 21:39:14,002 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/syswow64/d3d11 with DXVK version
INFO     2019-07-20 21:39:14,002 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/syswow64/d3d10core with DXVK version
INFO     2019-07-20 21:39:14,002 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/syswow64/d3d10_1 with DXVK version
INFO     2019-07-20 21:39:14,002 [dxvk.enable_dxvk_dll:131]:Replacing /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix/drive_c/windows/syswow64/d3d10 with DXVK version
DEBUG    2019-07-20 21:39:14,002 [system.execute:38]:Executing pgrep Steam.exe$
DEBUG    2019-07-20 21:39:14,011 [display._get_vidmodes:15]:Retrieving video modes from XrandR
DEBUG    2019-07-20 21:39:14,018 [game.configure_game:305]:Launching RESIDENT EVIL 2 / BIOHAZARD RE:2: {'env': {'DXVK_HUD': '0', 'STEAM_RUNTIME': '/home/laichiaheng/.local/share/lutris/runtime/steam', 'LD_LIBRARY_PATH': '/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/lib:/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/lib64:/lib:/lib64:/usr/lib:/usr/lib64:/usr/lib32:/usr/lib64:/home/laichiaheng/.local/share/lutris/runtime/Ubuntu-18.04-i686:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/lib:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/usr/lib:/home/laichiaheng/.local/share/lutris/runtime/Ubuntu-18.04-x86_64:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/lib:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/usr/lib:$LD_LIBRARY_PATH', 'WINEDEBUG': '-all', 'WINEARCH': 'win64', 'WINE': '/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine', 'WINEPREFIX': '/home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix', 'WINEESYNC': '1', 'WINEDLLOVERRIDES': 'mfplat=n,b;d3d10,d3d10_1,d3d10core,d3d11,dxgi=n'}, 'command': ['/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine', '/home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/Steam.exe', '-no-cef-sandbox', '-console', 'steam://rungameid/883710']}
DEBUG    2019-07-20 21:39:14,018 [game.configure_game:306]:Game info: {
  "env": {
    "DXVK_HUD": "0",
    "STEAM_RUNTIME": "/home/laichiaheng/.local/share/lutris/runtime/steam",
    "LD_LIBRARY_PATH": "/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/lib:/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/lib64:/lib:/lib64:/usr/lib:/usr/lib64:/usr/lib32:/usr/lib64:/home/laichiaheng/.local/share/lutris/runtime/Ubuntu-18.04-i686:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/lib:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/usr/lib:/home/laichiaheng/.local/share/lutris/runtime/Ubuntu-18.04-x86_64:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/lib:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/usr/lib:$LD_LIBRARY_PATH",
    "WINEDEBUG": "-all",
    "WINEARCH": "win64",
    "WINE": "/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine",
    "WINEPREFIX": "/home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix",
    "WINEESYNC": "1",
    "WINEDLLOVERRIDES": "mfplat=n,b;d3d10,d3d10_1,d3d10core,d3d11,dxgi=n"
  },
  "command": [
    "/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine",
    "/home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/Steam.exe",
    "-no-cef-sandbox",
    "-console",
    "steam://rungameid/883710"
  ]
}
DEBUG    2019-07-20 21:39:14,018 [command.start:111]:Running /usr/bin/lutris-wrapper 0 0 /home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/Steam.exe -no-cef-sandbox -console steam://rungameid/883710
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: SDL_VIDEO_FULLSCREEN_DISPLAY="off"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: DXVK_HUD="0"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: STEAM_RUNTIME="/home/laichiaheng/.local/share/lutris/runtime/steam"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: LD_LIBRARY_PATH="/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/lib:/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/lib64:/lib:/lib64:/usr/lib:/usr/lib64:/usr/lib32:/usr/lib64:/home/laichiaheng/.local/share/lutris/runtime/Ubuntu-18.04-i686:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/lib:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/i386/usr/lib:/home/laichiaheng/.local/share/lutris/runtime/Ubuntu-18.04-x86_64:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/lib:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/laichiaheng/.local/share/lutris/runtime/steam/amd64/usr/lib:$LD_LIBRARY_PATH"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: WINEDEBUG="-all"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: WINEARCH="win64"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: WINE="/home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: WINEPREFIX="/home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/RESIDENT EVIL 2  BIOHAZARD RE2/prefix"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: WINEESYNC="1"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: WINEDLLOVERRIDES="mfplat=n,b;d3d10,d3d10_1,d3d10core,d3d11,dxgi=n"
DEBUG    2019-07-20 21:39:14,018 [command.start:113]:ENV: LD_PRELOAD="/usr/$LIB/libgamemodeauto.so"
DEBUG    2019-07-20 21:39:14,019 [command.start:113]:ENV: PYTHONPATH="/usr/bin:/usr/lib/python37.zip:/usr/lib/python3.7:/usr/lib/python3.7/lib-dynload:/usr/lib/python3.7/site-packages"
DEBUG    2019-07-20 21:39:14,021 [signals.register_handler:26]:Registering <bound method MonitoredCommand.on_stop of <lutris.command.MonitoredCommand object at 0x7efbf9c4eef0>> to 7466
Running /home/laichiaheng/.local/share/lutris/runners/wine/tkg-protonified-4.6-x86_64/bin/wine /home/laichiaheng/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/Steam.exe -no-cef-sandbox -console steam://rungameid/883710
gamemodeauto: 
gamemodeauto: 
S:\Program Files (x86)\Steam\bin\cef\cef.win7x64\steamwebhelper.exe: invalid option -- `-n'
Try 'steamwebhelper.exe --help' for more information.
System page size: 4096
Could not find page for trampoline in +/- 2GB range of function to hook
Error allocating trampoline memory (no memory within +/-2gb? prior failures?)
[0720/213915.323:ERROR:registration_protocol_win.cc(84)] TransactNamedPipe: _޽uC (0x6D)
[0720/213915.325:INFO:crash_reporting.cc(242)] Crash reporting enabled for process: browser
[0720/213915.580:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 0
[0720/213915.888:ERROR:registration_protocol_win.cc(56)] CreateFile: 䤣ɮסC (0x2)
[0720/213915.888:INFO:crash_reporting.cc(242)] Crash reporting enabled for process: gpu-process
info:  Game: steamwebhelper.exe
info:  DXVK: v1.3-2
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
Xlib:  extension "NV-GLX" missing on display ":1".
info:  AMD RADV POLARIS10 (LLVM 8.0.0):
info:    Driver: 19.1.2
info:    Vulkan: 1.1.90
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:    Memory Heap[2]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Enabled device extensions:
info:    VK_EXT_conditional_rendering
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_memory_priority
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_dedicated_allocation
info:    VK_KHR_descriptor_update_template
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_get_memory_requirements2
info:    VK_KHR_image_format_list
info:    VK_KHR_maintenance1
info:    VK_KHR_maintenance2
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_draw_parameters
info:    VK_KHR_swapchain
info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 12 compiler threads
warn:  DXGI: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
warn:  D3D11DeviceContext::QueryInterface: Unknown interface query
warn:  b4e3c01d-e79e-4637-91b2-510e9f4c9b8f
err:   DXGI: CheckInterfaceSupport: Unsupported interface
err:   54ec77fa-1377-44e6-8c32-88fd5f44c84c
[0720/213916.000:ERROR:angle_platform_impl.cc(47)] populateRenderer11DeviceCaps(900): Error querying driver version from DXGI Adapter.
warn:  D3D11Device::GetDeviceRemovedReason: Stub
gamemodeauto: 
[0720/213918.843:ERROR:registration_protocol_win.cc(56)] CreateFile: 䤣ɮסC (0x2)
[0720/213918.844:INFO:crash_reporting.cc(242)] Crash reporting enabled for process: renderer
[0720/213919.200:ERROR:registration_protocol_win.cc(56)] CreateFile: 䤣ɮסC (0x2)
[0720/213919.200:INFO:crash_reporting.cc(242)] Crash reporting enabled for process: renderer
[0720/213919.427:WARNING:cert_verify_proc_win.cc(104)] Unknown error -2146762482 mapped to net::ERR_FAILED
wine: Unhandled page fault on read access to 0xffffffffffffffff at address 0x15330398d (thread 00ac), starting debugger...
gamemodeauto: 
0053:fixme:ntdll:EtwEventRegister ({47a9201e-73b0-42ce-9821-7e134361bc6f}, 0x18008b490, 0x1801603a8, 0x1801603a0) stub.
0053:fixme:ntdll:EtwEventRegister ({58a9201e-73b0-42ce-9821-7e134361bc70}, 0x18008b490, 0x1801603e0, 0x1801603d8) stub.
0053:fixme:ntdll:EtwEventRegister ({3fa9201e-73b0-43fe-9821-7e145359bc6f}, 0x18008b490, 0x180160370, 0x180160368) stub.
0053:fixme:ntdll:EtwEventRegister ({1432afee-73b0-42ce-9821-7e134361b433}, 0x18008b490, 0x180160418, 0x180160410) stub.
0053:fixme:ntdll:EtwEventRegister ({4372afee-73b0-42ce-9821-7e134361b519}, 0x18008b490, 0x180160450, 0x180160448) stub.
0053:fixme:imm:ImmGetOpenStatus (0x5db00): semi-stub
Unhandled exception: page fault on read access to 0xffffffffffffffff, invalid program stack in 64-bit code (0x000000015330398d).
Register dump:
 rip:000000015330398d rsp:0000000000527348 rbp:000000014ea28b6d eflags:00010206 (  R- --  I   - -P- )
 rax:000000015330398d rbx:ffffffffc8a3042f rcx:0000000149c9e2ba rdx:ffffffffb220beb0
 rsi:0000000000527500 rdi:ffffffff00000000  r8:0000000149c9e2ba  r9:0000000000000000 r10:0000000000000000
 r11:0000000020b8ff26 r12:0000000000000000 r13:0000000000000000 r14:0000000000000005 r15:0000000027225af9
Stack dump:
0x0000000000527348:  0000000000527348 0000000000111185
0x0000000000527358:  0000000000000000 0000000000000000
0x0000000000527368:  0000000000000000 0000000000000000
0x0000000000527378:  0000000000000000 0000000000000000
0x0000000000527388:  0000000000000000 0000000000000000
0x0000000000527398:  0000000000000000 0000000000000000
0x00000000005273a8:  0000000000000000 0000000000000000
0x00000000005273b8:  0000000000000000 0000000000000000
0x00000000005273c8:  0000000000000000 0000000000000000
0x00000000005273d8:  0000000000000000 0000000000000000
0x00000000005273e8:  0000000000000000 0000000000000000
0x00000000005273f8:  0000000000000000 0000000000000000
Backtrace:
=>0 0x000000015330398d EntryPoint+0xfcb1f96d() in re2 (0x000000014ea28b6d)
0x000000015330398d EntryPoint+0xfcb1f96d in re2: sgdt	0xfffffffff701dbd1(%rip)
Modules:
Module	Address					Debug info	Name (63 modules)
PE	         1ea0000-         2037000	Deferred        dxgi
PE	        6a340000-        6a5a7000	Deferred        d3d11
PE	        6ecc0000-        6f92b000	Deferred        amd_ags_x64
PE	        7b420000-        7b5e0000	Deferred        kernel32
PE	        7bc20000-        7bc33000	Deferred        ntdll
PE	       140000000-       156825000	Export          re2
PE	       180000000-       1801a8000	Deferred        gameoverlayrenderer64
PE	     7ff33e70000-     7ff33eb2000	Deferred        mfreadwrite
PE	     7ff385d0000-     7ff3863d000	Deferred        mfplat
PE	    7efbfab10000-    7efbfab17000	Deferred        winevulkan
PE	    7efbfab60000-    7efbfab64000	Deferred        api-ms-win-crt-locale-l1-1-0
PE	    7efbfab70000-    7efbfab77000	Deferred        vulkan-1
PE	    7efbfab90000-    7efbfabae000	Deferred        xinput1_3
PE	    7efbfabb0000-    7efbfabb4000	Deferred        avrt
PE	    7efbfabe0000-    7efbfabf0000	Deferred        msvcrt
PE	    7efbfadc0000-    7efbfadc6000	Deferred        iphlpapi
PE	    7efbfadf0000-    7efbfadf5000	Deferred        hid
PE	    7efbfae10000-    7efbfae1c000	Deferred        dinput8
PE	    7efbfae80000-    7efbfae84000	Deferred        api-ms-win-core-sysinfo-l1-2-1
PE	    7efbfb0c0000-    7efbfb0c4000	Deferred        api-ms-win-core-localization-l1-2-1
PE	    7efbfb0d0000-    7efbfb0d4000	Deferred        api-ms-win-core-fibers-l1-1-1
PE	    7efbfb0e0000-    7efbfb0e4000	Deferred        api-ms-win-core-synch-l1-2-0
PE	    7efbfb100000-    7efbfb105000	Deferred        psapi
PE	    7efbfb130000-    7efbfb135000	Deferred        uxtheme
PE	    7efbfc0f0000-    7efbfc0f5000	Deferred        winex11
PE	    7efbfc1b0000-    7efbfc22c000	Deferred        winmm
PE	    7efbfc290000-    7efbfc29d000	Deferred        mpr
PE	    7efbfc2d0000-    7efbfc2f1000	Deferred        wininet
PE	    7efbfc380000-    7efbfc386000	Deferred        ws2_32
PE	    7efbfc3c0000-    7efbfc3c6000	Deferred        jsproxy
PE	    7efbfc3f0000-    7efbfc3f7000	Deferred        winhttp
PE	    7efbfc440000-    7efbfc456000	Deferred        vcruntime140
PE	    7efbfc450000-    7efbfc456000	Deferred        imm32
PE	    7efbfc480000-    7efbfc484000	Deferred        usp10
PE	    7efbfc4e0000-    7efbfc534000	Deferred        comctl32
PE	    7efbfc660000-    7efbfc66a000	Deferred        aclui
PE	    7efbfc680000-    7efbfc685000	Deferred        shcore
PE	    7efbfc6c0000-    7efbfc6cf000	Deferred        shlwapi
PE	    7efbfc770000-    7efbfd068000	Deferred        shell32
PE	    7efbfd1e0000-    7efbfd1f7000	Deferred        setupapi
PE	    7efbfd290000-    7efbfd29d000	Deferred        oleaut32
PE	    7efbfd420000-    7efbfd42b000	Deferred        rpcrt4
PE	    7efbfd4d0000-    7efbfd4d5000	Deferred        version
PE	    7efbfd520000-    7efbfd611000	Deferred        user32
PE	    7efbfd7b0000-    7efbfd7dc000	Deferred        ole32
PE	    7efbfd9a0000-    7efbfda3c000	Deferred        msvcp140
PE	    7efbfdab0000-    7efbfdabc000	Deferred        gdi32
PE	    7efbfde60000-    7efbfde67000	Deferred        dbghelp
PE	    7efbfdf00000-    7efbfdf19000	Deferred        msvcr120
PE	    7efbfe050000-    7efbfe078000	Deferred        msvcp120
PE	    7efbfe140000-    7efbfe191000	Deferred        concrt140
PE	    7efbfe160000-    7efbfe164000	Deferred        api-ms-win-crt-utility-l1-1-0
PE	    7efbfe180000-    7efbfe184000	Deferred        api-ms-win-crt-time-l1-1-0
PE	    7efbfe190000-    7efbfe195000	Deferred        api-ms-win-crt-string-l1-1-0
PE	    7efbfe1b0000-    7efbfe1b5000	Deferred        api-ms-win-crt-stdio-l1-1-0
PE	    7efbfe1c0000-    7efbfe1c5000	Deferred        api-ms-win-crt-runtime-l1-1-0
PE	    7efbfe1d0000-    7efbfe1d6000	Deferred        api-ms-win-crt-math-l1-1-0
PE	    7efbfe1f0000-    7efbfe1f4000	Deferred        api-ms-win-crt-heap-l1-1-0
PE	    7efbfe200000-    7efbfe204000	Deferred        api-ms-win-crt-filesystem-l1-1-0
PE	    7efbfe240000-    7efbfe25e000	Deferred        ucrtbase
PE	    7efbfe360000-    7efbfe365000	Deferred        api-ms-win-crt-convert-l1-1-0
PE	    7efbfe380000-    7efbfe38b000	Deferred        advapi32
PE	    7efbfe960000-    7efbfe963000	Deferred        wow64cpu
Threads:
process  tid      prio (all id:s are in hex)
0000000e services.exe
	[C:\windows\system32\services.exe]
	00000024    0
	00000021    0
	0000001c    0
	00000018    0
	00000013    0
	00000010    0
	0000000f    0
00000011 winedevice.exe
	[C:\windows\system32\winedevice.exe]
	00000019    0
	00000017    0
	00000016    0
	00000012    0
0000001a plugplay.exe
	[C:\windows\system32\plugplay.exe]
	0000001e    0
	0000001d    0
	0000001b    0
0000001f winedevice.exe
	[C:\windows\system32\winedevice.exe]
	00000025    0
	00000023    0
	00000022    0
	00000020    0
000000e3 explorer.exe
	[C:\windows\system32\explorer.exe /desktop]
	000000e9    0
	000000e8    0
	000000e7    0
	000000e4    0
000001f0 Steam.exe
	["S:\Program Files (x86)\Steam\Steam.exe" -no-cef-sandbox -console steam://rungameid/883710]
	0000008a    0
	000000c9    0
	000000f4    0
	000000d1    0
	00000097    0
	00000084    0
	00000015    0
	00000073    0
	00000048    0
	00000036    0
	00000037    0
	0000004c    0
	00000038    0
	0000003a    0
	0000003b    0
	0000000a    0
	0000000d    0
	0000000c    2
	00000205    0
	00000203    2
	00000202    0
	00000201    0
	000001fd    0
	000001fc    0
	000001f9    0
	000001f8    0
	000001f7    0
	000001f1    0
000001f2 explorer.exe
	[C:\windows\system32\explorer.exe /desktop]
	000001f6    0
	000001f5    0
	000001f4    0
	000001f3    0
000001fa steamwebhelper.exe
	["S:\Program Files (x86)\Steam\bin\cef\cef.win7x64\steamwebhelper.exe" "-lang=zh_TW" "-cachedir=C:\users\laichiaheng\Local Settings\Application Data\Steam\htmlcache" "-steampid=496" "-buildid=1563346602" "-steamid=0" "-steamuniverse=Dev" "-clientui=S:\Program Files (x86)\Steam\clientui" --enable-blink-features=ResizeObserver,Worklet,AudioWorklet --enable-media-stream --enable-smooth-scrolling --num-raster-threads=4 --enable-direct-write --disablehighdpi --force-device-scale-factor=1 --device-scale-factor=1 --no-sandbox "--log-file=S:\Program Files (x86)\Steam\logs\cef_log.txt" --no-sandbox]
	0000008c    0
	0000008b    0
	000000b9    0
	000000d2    0
	00000098    0
	000000e2    0
	000000cb    0
	000000eb    0
	000000e6    0
	000000e0   -2
	000000de    0
	000000da    0
	000000db    0
	000000c1    0
	000000b4    0
	000000b3   -2
	00000095    0
	00000090    0
	00000091    0
	0000003e    0
	00000074   -2
	00000081    0
	00000080    0
	0000007f    0
	0000007e    0
	0000007d    0
	0000007c    0
	0000007b    0
	0000007a    0
	00000079   -2
	00000078   -2
	00000077    0
	00000076    0
	00000075    0
	00000083    0
	000001fb    0
0000009b steamwebhelper.exe
	["S:\Program Files (x86)\Steam\bin\cef\cef.win7x64\steamwebhelper.exe" --type=gpu-process --field-trial-handle=560,5738000990510850424,1692542361099340912,131072 --no-sandbox --log-file="S:\Program Files (x86)\Steam\logs\cef_log.txt" --product-version="Valve Steam Client" --lang=zh-TW --force-device-scale-factor=1 --disablehighdpi --buildid=1563346602 --steamid=0 --gpu-preferences=KAAAAAAAAACAAwCAAQAAAAAAAAAAAGAAAAAAAAAAAAAIAAAAAAAAACgAAAAEAAAAIAAAAAAAAAAoAAAAAAAAADAAAAAAAAAAOAAAAAAAAAAQAAAAAAAAAAAAAAAFAAAAEAAAAAAAAAAAAAAABgAAABAAAAAAAAAAAQAAAAUAAAAQAAAAAAAAAAEAAAAGAAAA --service-request-channel-token=8910068116780664375 --mojo-platform-channel-handle=580 /prefetch:2 --no-sandbox]
	000000b2    0
	000000be    0
	000000d8    0
	000000d7    0
	000000d5    0
	00000065    0
	000000f1   -2
	000000b7    0
	000000f2    0
	000000ee    0
	000000f3   -2
	0000005b   -2
	0000008d    0
	00000105    0
	00000104    0
	000000ca    0
	0000009c    0
000000d6 steamwebhelper.exe
	["S:\Program Files (x86)\Steam\bin\cef\cef.win7x64\steamwebhelper.exe" --type=renderer --no-sandbox --force-device-scale-factor=1 --field-trial-handle=560,5738000990510850424,1692542361099340912,131072 --service-pipe-token=11225351955323019840 --enable-blink-features=ResizeObserver,Worklet,AudioWorklet --lang=zh-TW --log-file="S:\Program Files (x86)\Steam\logs\cef_log.txt" --product-version="Valve Steam Client" --force-device-scale-factor=1 --disablehighdpi --buildid=1563346602 --steamid=0 --device-scale-factor=1 --num-raster-threads=4 --enable-main-frame-before-activation --service-request-channel-token=11225351955323019840 --renderer-client-id=3 --mojo-platform-channel-handle=900 /prefetch:1 --no-sandbox]
	00000087    0
	00000086    0
	00000085    0
	00000071    0
	00000062    0
	000000af    0
	0000008f    0
	0000008e    0
	000000c4    0
	000000c3    0
	000000b8   -2
	000000b6    0
	000000b5    0
	000000bd    0
	000000bc    0
	00000059    0
	000000cd    0
	000000cc    0
	0000005a    0
	00000050    0
	000000d0   -2
	0000006a   -2
	00000109    0
	0000010a    0
	0000009f    0
	000000d9    0
	00000107    1
000000ba steamwebhelper.exe
	["S:\Program Files (x86)\Steam\bin\cef\cef.win7x64\steamwebhelper.exe" --type=renderer --no-sandbox --force-device-scale-factor=1 --field-trial-handle=560,5738000990510850424,1692542361099340912,131072 --disable-gpu-compositing --service-pipe-token=5915005609417236175 --enable-blink-features=ResizeObserver,Worklet,AudioWorklet --lang=zh-TW --log-file="S:\Program Files (x86)\Steam\logs\cef_log.txt" --product-version="Valve Steam Client" --force-device-scale-factor=1 --disablehighdpi --buildid=1563346602 --steamid=0 --device-scale-factor=1 --num-raster-threads=4 --enable-main-frame-before-activation --service-request-channel-token=5915005609417236175 --renderer-client-id=4 --mojo-platform-channel-handle=1072 /prefetch:1 --no-sandbox]
	000000aa    0
	000000a9   -2
	000000a7    0
	000000a6    0
	000000a2    0
	000000a1    0
	000000a0    0
	0000009e    0
	0000009d    0
	0000005c    0
	000000a5    0
	000000a4   -2
	0000006e   -2
	000000a8    0
	00000030    0
	000000b1    0
	000000f0    0
	000000bb    0
000000ab (D) S:\Program Files (x86)\Steam\steamapps\common\RESIDENT EVIL 2  BIOHAZARD RE2\re2.exe
	["S:\Program Files (x86)\Steam\steamapps\common\RESIDENT EVIL 2  BIOHAZARD RE2\re2.exe"]
	000000ac    0 <==
System information:
    Wine build: wine-4.6.r0.g3f8edce5 ( TkG Staging Esync )
    Platform: x86_64
    Version: Windows 7
    Host system: Linux
    Host version: 5.2.1-1-MANJARO
0053:fixme:imm:ImmReleaseContext ((nil), (nil)): stub
0053:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0053:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0053:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0053:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0053:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
[0720/213930.869:ERROR:registration_protocol_win.cc(56)] CreateFile: 䤣ɮסC (0x2)
[0720/213930.869:INFO:crash_reporting.cc(242)] Crash reporting enabled for process: gpu-process
Xlib:  extension "NV-GLX" missing on display ":1".
System page size: 4096
Could not find page for trampoline in +/- 2GB range of function to hook
Error allocating trampoline memory (no memory within +/-2gb? prior failures?)
System page size: 4096
Could not find page for trampoline in +/- 2GB range of function to hook
Error allocating trampoline memory (no memory within +/-2gb? prior failures?)
IPC client is in my process, could/should be using an in-process pipe.IPC client is in my process, could/should be using an in-process pipe.IPC client is in my process, could/should be using an in-process pipe.ILocalize::AddFile() failed to load file "public/steambootstrapper_english.txt".
ILocalize::AddFile() failed to load file "public/steambootstrapper_tchinese.txt".
[2019-07-20 21:39:14] Startup - updater built Jul 16 2019 23:00:31
[2019-07-20 21:39:14] Verifying installation...
[2019-07-20 21:39:14] Verification complete
[2019-07-20 21:39:15] Background update loop checking for update. . .
[2019-07-20 21:39:15] Checking for available update...
[2019-07-20 21:39:15] Downloading manifest: client-download.steampowered.com/client/steam_client_win32
[2019-07-20 21:39:16] Download skipped by HTTP 304 Not Modified
[2019-07-20 21:39:16] Nothing to do
[2019-07-20 21:39:38] Shutdown
Waiting on children
All children gone
Exit with returncode 0
DEBUG    2019-07-20 21:39:38,754 [command.on_stop:158]:The process has terminated with code 0
DEBUG    2019-07-20 21:39:38,754 [command.stop:241]:Management process looks dead already.
DEBUG    2019-07-20 21:39:38,754 [command.stop:249]:Detaching logger
DEBUG    2019-07-20 21:39:40,034 [game.beat:585]:Game thread stopped
WARNING  2019-07-20 21:39:40,035 [game.on_game_quit:613]:Game still running (state: running)
INFO     2019-07-20 21:39:40,035 [game.stop:596]:Stopping RESIDENT EVIL 2 / BIOHAZARD RE:2 (winesteam)
DEBUG    2019-07-20 21:39:40,035 [command.stop:241]:Management process looks dead already.
DEBUG    2019-07-20 21:39:40,035 [command.stop:253]:logger already detached
DEBUG    2019-07-20 21:39:40,042 [game.on_game_quit:628]:RESIDENT EVIL 2 / BIOHAZARD RE:2 stopped at 六, 20  7月 2019 21:39:40
DEBUG    2019-07-20 21:39:40,042 [game.save:212]:Saving RESIDENT EVIL 2 / BIOHAZARD RE:2 (winesteam)
DEBUG    2019-07-20 21:39:40,050 [lutriswindow.on_game_updated:698]:Updating game RESIDENT EVIL 2 / BIOHAZARD RE:2 (winesteam)

INFO     2019-07-20 21:39:47,750 [application.do_shutdown:440]:Shutting down Lutris
```

**System information**
[lutris-issue-report.zip](https://github.com/lutris/lutris/files/3413667/lutris-issue-report.zip)

**Screenshots**

No.

**Comment by tannisroot** [comment]
Try disabling DXVK.

**Comment by laichiaheng** [comment]
@tannisroot It's known as a AMD CPU or WINE bug.
https://github.com/ValveSoftware/Proton/issues/2927#issuecomment-519363498

**Comment by tannisroot** [comment]
Oh, closing this is NOTOURBUG then.
